src/main/kotlin/calculator/Splitter.kt

```diff
@@ -1,20 +1,18 @@
 package calculator
 
-class Splitter {
-    companion object {
-        fun getSymbols(source: String): List<String> =
-            source.split(" ")
-                .filter {
-                    it.toIntOrNull() === null
-                }
+object Splitter {
+    fun getSymbols(source: String): List<String> =
+        source.split(" ")
+            .filter {
+                it.toIntOrNull() === null
+            }
 
-        fun getNumbers(source: String): List<Int> =
-            source.split(" ")
-                .filter {
-                    it.toIntOrNull() !== null
-                }
-                .map {
-                    it.toInt()
-                }
-    }
+    fun getNumbers(source: String): List<Int> =
+        source.split(" ")
+            .filter {
```

Comment:
```suggestion
            .filterNot {
```
filterNot을 활용할 수 있어요